app/assets/javascripts/admin/views/auth/auth_token_view.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React from "react";
+import { Form, Input, Button, Col, Row, Spin } from "antd";
+import Request from "libs/request";
+
+const FormItem = Form.Item;
+
+type State = {
+  isLoading: boolean,
+  currentToken: string,
+};
+
+class AuthTokenView extends React.PureComponent<{}, State> {
+  state = {
+    isLoading: true,
+    currentToken: "",
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData(): Promise<void> {
+    const { token } = await Request.receiveJSON("/api/auth/token");
+
+    this.setState({
+      isLoading: false,
+      currentToken: token,
+    });
+  }
+
+  async revokeToken(): Promise<void> {
+    this.setState({ isLoading: true });
+    try {
+      await Request.triggerRequest("/api/auth/token", { method: "DELETE" });
```

Comment:
same here. I also believe that we don't use anywhere outside of the `Request`lib. Perhaps `receiveJSON`will do?